app/frontend/src/features/connections/friends/FriendSummaryView.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+import { Link } from "react-router-dom";
+import TextBody from "../../../components/TextBody";
+import { User } from "../../../pb/api_pb";
+
+const useStyles = makeStyles((theme) => ({
+  friendItem: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    padding: `0 ${theme.spacing(1)}`,
+  },
+  friendLink: {
+    color: theme.palette.text.primary,
+    textDecoration: "none",
+  },
+  userLoadErrorAlert: {
+    borderRadius: 0,
+  },
+}));
+
+interface FriendSummaryViewProps {
+  children?: React.ReactNode;
+  friend?: User.AsObject;
+}
+
+function FriendSummaryView({ children, friend }: FriendSummaryViewProps) {
+  const classes = useStyles();
+
+  return friend ? (
+    <Box className={classes.friendItem} key={friend.userId}>
+      <Link className={classes.friendLink} to={`/user/${friend.username}`}>
+        <Typography variant="h2" component="h3">
```

Comment:
I think it's better to use a className, but I don't feel super strongly about it.